doc/src/sphinx/Quickstart.rst

```diff
@@ -22,6 +22,7 @@ Setting up SBT
 
 We'll use sbt_ to build our project. Finagle is published to Maven Central,
 so little setup is needed: see the ``build.sbt`` in the ``quickstart`` directory.
+Also ensure that you're using ``scala`` ``2.11`` as of ``6.36.0``.
```

Comment:
It might be better to phrase this as "we no longer support 2.10" since we plan on adding 2.12 support when it comes out (which is hopefully soon).